test/Product_DelimitedTable_INVALID_lid_ref_no-match.xml

```diff
@@ -0,0 +1,282 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://pds.nasa.gov/pds4/pds/v1/PDS4_PDS_1K00.sch"
+  schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<?xml-model href="http://pds.nasa.gov/pds4/proc/v1/PDS4_PROC_1K00_1300.sch"
+    schematypens="http://purl.oclc.org/dsdl/schematron"?>
+
+<Product_Observational xmlns="http://pds.nasa.gov/pds4/pds/v1"
+    xmlns:pds="http://pds.nasa.gov/pds4/pds/v1"
+    xmlns:proc="http://pds.nasa.gov/pds4/proc/v1"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://pds.nasa.gov/pds4/pds/v1 https://pds.nasa.gov/pds4/pds/v1/PDS4_PDS_1K00.xsd
+                        http://pds.nasa.gov/pds4/proc/v1 https://pds.nasa.gov/pds4/proc/v1/PDS4_PROC_1K00_1300.xsd">  
+    
+    <Identification_Area>
+        <logical_identifier>urn:nasa:pds:example.dph.sampleproducts:exampleproducts:streamdelimited.co-s-inst-2-dummy-data-v1.0</logical_identifier>
+        <version_id>1.0</version_id>
+        <title>CASSINI Orbiter Instrument Experiment</title>
+        <information_model_version>1.20.0.0</information_model_version>

```

Comment:
@rsjoyner per the [Regression Test Data Rules](https://pds-data-dictionaries.github.io/development/ldd-test.html?highlight=test#regression-test-data-rules), there are a few quirks to the system we need to keep in mind. Here are links to examples of what those rules are referring to:
* No schematron: https://github.com/pds-data-dictionaries/ldd-template/blob/main/test/test1_FAIL.xml#L3
* No schema location: https://github.com/pds-data-dictionaries/ldd-template/blob/main/test/test1_FAIL.xml#L10
* IM Version "variable": https://github.com/pds-data-dictionaries/ldd-template/blob/main/test/test1_FAIL.xml#L16